database/sql/V027__quotes_adding_onchain_signing_scheme.sql

```diff
@@ -0,0 +1,6 @@
+-- Adding new column on table quotes to differentiate the two on-chain signing schemes
+-- This is required as different signing schemes have different expirations
+CREATE TYPE OnchainSigningScheme AS ENUM ('eip1271', 'presign');
```

Comment:
(Nits, feel free to ignore.)
I imagine this will be reused to add different behavior to different quote types in the future if needed (e.g., ETH flow orders have a longer expiration for whatever reason).
Then I'd rename `OnchainSigningScheme` to `QuoteType` just to point out that the signing scheme has nothing to do with this entry (except that it so happens that the two types we have now correspond to the two signature types).
We could also have a different order type for normal quotes instead of making the field `NULL`able (maybe `eoa`?).

Reply:
I think this is a fair suggestion. Maybe "OnchainQuoteType"?

Reply:
I kinda like that Federico suggests making it even more general, not just for onchain orders. 
I would call it `QuoteType` and introduce a new enum value `eoa`. `eoa` would then also be the default value.

This would also remove the "weird" `IS NOT DISTINCT FROM` syntax.

Reply:
I wouldn't call it `eoa` since, techincally, you can place EIP-1271 and Presign orders as well.

Maybe `standard`?